ripley/api/auth_controller.py

```diff
@@ -101,13 +101,17 @@ def lti_launch():
         launch_data_storage = FlaskCacheDataStorage(cache)
 
         message_launch = MessageLaunch(flask_request, tool_conf, launch_data_storage=launch_data_storage)
-        message_launch_data = message_launch.get_launch_data()
-        custom_fields = message_launch_data.get('https://purl.imsglobal.org/spec/lti/claim/custom', {})
-        uid = custom_fields.get('canvas_user_login_id')
-        canvas_api_domain = custom_fields.get('canvas_api_domain')
+        launch_data = message_launch.get_launch_data()
+        canvas_api_domain = _get_custom_param(launch_data, 'canvas_api_domain', is_numeric=False)
```

Comment:
@lyttam - do we need `canvas_api_domain`?

Reply:
@johncrossman no, I'll remove it.